config/rbac/role_binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: manager-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: manager-role
+subjects:
+- kind: ServiceAccount
+  name: default
```

Comment:
🤔 does this get automatically rewritten to be the name of the service account?

Reply:
Yes, although how kustomize decides whether to transform these things is still unclear to me:

```
kubectl describe clusterrolebinding signer-ca-manager-rolebinding
Name:         signer-ca-manager-rolebinding
Labels:       <none>
Annotations:  kubectl.kubernetes.io/last-applied-configuration:
                {"apiVersion":"rbac.authorization.k8s.io/v1","kind":"ClusterRoleBinding","metadata":{"annotations":{},"name":"signer-ca-manager-rolebindin...
Role:
  Kind:  ClusterRole
  Name:  signer-ca-manager-role
Subjects:
  Kind            Name     Namespace
  ----            ----     ---------
  ServiceAccount  default  signer-ca-system

```